elegant/gui/experiment_annotator.py

```diff
@@ -11,7 +11,7 @@
 from .. import load_data
 
 class ExperimentAnnotator:
-    def __init__(self, ris_widget, experiment_name, positions, annotation_fields, start_position=None, readonly=False):
+    def __init__(self, ris_widget, experiment_name, positions, annotation_fields, start_position=None, readonly=False, annotation_subdir='annotations'):
```

Comment:
I have a mild preference for `annotation_dir` vs. `annotation_subdir` just because it's fewer characters (and the fact that it's called `experiment_root` is probably enough to suggest that the annotation dir is a subdirectory of the root...). Also don't forget to document this parameter in the relevant docstrings.